web_widget_open_tab/demo/res_users_view.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_users_tree" model="ir.ui.view">
+        <field name="model">res.users</field>
+        <field name="inherit_id" ref="base.view_users_tree" />
+        <field name="arch" type="xml">
+            <field name="name" position="before">
+                <field name="id" widget="open_tab" />
```

Comment:
Could you add in the roadmap/limitations that the module currently does not support this:
```
                <field name="company_id" position="attributes">
                     <attribute name="widget">open_tab</attribute>
                </field>
```

For the time being this would still open the user form.

Reply:
For this the module https://github.com/OCA/web/tree/13.0/web_tree_many2one_clickable should be used

Reply:
Indeed, yet that module does not currently allow opening in a new tab.
It could be added to the Roadmap of the web_tree_many2one_clickable indeed as an option. 
So out of the scope if this MR.

It's kind of strange that we can add the widget on any field, and always have the same behavior, but I guess it's acceptable.